src/components/svgImage/index.web.tsx

```diff
@@ -31,7 +30,7 @@ function SvgImage(props: SvgImageProps) {
   };
 
   if (isSvgUri(data)) {
-    return <img {...others} src={data.uri} style={styleObj}/>;
+    return <img {...others} src={data.uri} style={StyleSheet.flatten(style)}/>;
```

Comment:
I think that the flatter is not necessary here.. only for the postCSS thing

Reply:
we need that, its not supported on React

tested in: 
https://playcode.io/react
```
import React from 'react';

export function App(props) {
  return (
    <div className='App'>
      <img src="https://reactjs.org/logo-og.png" alt="react logo" style={[{ width: '400px'}]}/>
    </div>
  );
}
```